packages/ant-component-mapper/src/common/form-wrapper.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Form } from 'antd';
+
+const AntForm = ({ label, children, layout, component, invalid, isRequired, help }) => {
+  return (
+    <Form layout={layout} component={component}>
+      <Form.Item
```

Comment:
Note: We have to get rid of this somehow (I am afraid that we will have to mock this component), but in some future PR.